data/03-09-random-forest.php

```diff
@@ -44,14 +44,62 @@
 			?>
 		</header>
 
-		<section class="col-xs-22 col-xs-offset-1 col-sm-18 col-sm-offset-3 col-md-14 col-md-offset-5 col-lg-10 col-lg-offset-7">
-			<p>O algoritmo naïve Bayes (NB) é um classificador fundamentado em diversos conceitos da Teoria da Probabilidade, como probabilidade condicional, independência condicional, regra da multiplicação, distribuição conjunta de probabilidades e, especialmente, em um importante teorema chamado Teorema de Bayes. Uma característica atraente deste classificador é a sua capacidade de produzir diretamente estimativas de probabilidade para cada rótulo de classe em vez de simples classificações. Além disso, o classificador se destaca por seu baixo custo computacional na etapa de treinamento, o que significa que ele gera o modelo de classificação em um tempo mais rápido em comparação a outros algoritmos. As subseções a seguir descrevem as etapas de treinamento e classificação do NB.</p>
-
-			<h2>Etapa de treinamento</h2>
-			<p>O princípio básico do NB consiste em treinar um modelo que possibilite a aplicação do <a href="https://pt.wikipedia.org/wiki/Teorema_de_Bayes"><strong>Teorema de Bayes</strong> (ou regra de Bayes)</a> para estimar a classe mais provável de um novo objeto. Para tal, na etapa de treinamento o algoritmo computa uma tabela de probabilidades que resume a base de dados de treinamento com informações suficientes para a aplicação deste teorema.</p>
-			<p>A seguir apresentaremos um exemplo que mostra detalhadamente o processo de treinamento de um classificador NB. Considere a base de dados apresentada na Tabela 4. Suponha que as 15 observações desta base representem dados de uma pesquisa sobre o endividamento de casais jovens. A base registra se o casal possui filhos, a escolaridade do(a) responsável pela família e se a família possui ou não algum tipo de dívida com o cartão de crédito.</p>
-
-			<!-- Tabela [inicio] -->
+		<div class="col-xs-22 col-xs-offset-1 col-sm-18 col-sm-offset-3 col-md-14 col-md-offset-5 col-lg-10 col-lg-offset-7">
```

Comment:
Não tem nada errado em usar a div aqui.
Eu passei a usar section pq chei mais simples ver onde a coluna começa e termina.
Quando começa a ter mais conteúdo, pode ficar meio embolado pq vc passa ter diversas `div`s pelo código.

```suggestion
		<section class="col-xs-22 col-xs-offset-1 col-sm-18 col-sm-offset-3 col-md-14 col-md-offset-5 col-lg-10 col-lg-offset-7">
```